bin/internal/dart-sdk.version

```diff
@@ -1 +1 @@
-1.25.0-dev.7.0
+hash/a83a69ff3eff845980308779cadbdb80cc03a603
```

Comment:
> (unless we add backwards compatibility to the format)

AFAIU what we discussed yesterday offline was that it probably isn't a good idea to move forward without this.